### PR TITLE
Do not create duplicate rules for duplicate media queries

### PR DESCRIPTION
**Pull request by bobjflong**

### CODE REVIEW

**Comment by bobjflong** [comment]
Travis: https://travis-ci.org/premailer/css_parser/builds/410945785?utm_source=github_status&utm_medium=notification

File: test/test_css_parser_media_types.rb [hunk 1]
```diff
@@ -8,6 +8,25 @@ def setup
     @cp = Parser.new
   end
 
+  def test_that_duplicate_media_queries_do_not_duplicate_rules
+    query_to_duplicate = ["only screen and (max-width: 480px)"]*100
```
Comment (jsteiner) [inline test/test_css_parser_media_types.rb:12]: Is it necessary to do this 100 times, or would 2 do the trick?

**Review by jsteiner — APPROVED** [review]

**Review by emb78 — APPROVED** [review]
Makes sense 👍

File: test/test_css_parser_media_types.rb (hunk 1 above)
Comment (bobjflong) [reply test/test_css_parser_media_types.rb:12]: 2 would do the trick. I think it makes no difference though. I will follow up in a bit

**Pull request merged**